### ISSUE TITLE
aicodereview doesnt work

### ISSUE THREAD

**Issue by kabaneridev** [kabaneridev/ottaJobsTelegramNotifier#2]
tldr; it didn't performed any review yet but the limit is gone? first of all, it will require research how the limits works, what are the limits, is there any free plan to utilize (it doesnt need to be paid) etc...

current issue:
https://github.com/kabaneridev/ottaJobsTelegramNotifier/actions/runs/8956117267/job/24597540832
<img width="1671" alt="Screenshot 2024-05-05 at 07 57 32" src="https://github.com/kabaneridev/ottaJobsTelegramNotifier/assets/48051231/ee54ea76-5f51-48dd-9d0d-f275d842b952">

expected: once having proper model set in workflow file, it should perform review once providing proper API Key
actual: limit is being somehow hit, code review is not being done...

**Comment by anyeor** [comment]
The limits for paid versions is 40 messages per 3 hours.

![image](https://github.com/kabaneridev/ottaJobsTelegramNotifier/assets/49734100/95c02800-155f-4149-9756-e4d9436e8ea7)

Read more here: https://help.openai.com/en/articles/6950777-what-is-chatgpt-plus

**Comment by kabaneridev** [comment]
https://community.openai.com/t/understanding-api-limits-and-free-tier/498517/6

so in order to get credits -> credit card needs to be added, but paying at this point for this make no sense, free alternative -> https://github.com/reviewdog/reviewdog
